GoogleForCreators/web-stories-wp
# Updated advanced dropdown search behavior

## Summary

Updated (font) dropdown search behavior so that:

* When searching, only a single group of "search results" is displayed (instead of groups of "recently used" and "recommended")
* Only items matching the search term as a prefix are shown, not just substring matches.

Quick demo:
![fontpicker_searchresults](https://user-images.githubusercontent.com/637548/99848448-ce5ff680-2b47-11eb-991e-45f5a9e2aed7.gif)

## To-do

* [x] Add new search results label
* [x] Handle search results by itself
* [x] Update search comparison

## Testing Instructions

1. Add two text fields
1. In the first text field, assign the font "Montserrat"
1. In the second text field, open the font dropdown and search for _"mon"_
1. Observe that only a single group of results are returned labeled _"search results"_
1. Also observe that all search results start with _"mon"_ and not just contain _"mon"_

---



Fixes #4008
